deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,23 @@
+allow-other: true
+
+logging:
+  type: base
+  level: log_debug
+
+components:
+  - libfuse
+  - file_cache
+  - attr_cache
+  - azstorage
+
+libfuse:
+  attribute-expiration-sec: 120
+  entry-expiration-sec: 120
+  negative-entry-expiration-sec: 240
+
+file_cache:
+  timeout-sec: 120
+  max-size-mb: 4096
```

Comment:
By putting max-size-mb limit we are saying temp cache of blobfuse is limited to 4GB only. By default in blobfusev1 there is no such limits unless user explicitly wants to contain the size. For symmetry with v1 we can just remove this config from here.